src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -63,12 +70,78 @@ def send_incident_alert_notification(
     }
 
     client = SlackClient(integration_id=integration.id)
+    parent_notification_message = None
     try:
-        client.post("/chat.postMessage", data=payload, timeout=5)
-        return True
-    except ApiError:
+        parent_notification_message = repository.get_parent_notification_message(
```

Comment:
We could! Getting the parent won't mean we're going to make it into a thread, so I thought it would be okay to just keep this logic. I can move it under the flag though if we want to keep all the changes under the flag

Reply:
Let's move it under the flag because one reason for feature flag is that if we break something we can turn it off quickly.

Reply:
Yeah moved everything under the flag!